django_tests/test_cloudinaryField.py

```diff
@@ -58,6 +59,8 @@ def test_pre_save(self):
         self.assertEqual(".png", os.path.splitext(prep_value)[1])
         self.assertEqual(TEST_IMAGE_W, p.image_width)
         self.assertEqual(TEST_IMAGE_H, p.image_height)
+        self.assertEqual(upload_mock.call_args[1]['phash'], 'true')
```

Comment:
There are 2 functions that need to be implemented:
    def test_upload_options_with_filename(self):
    def test_upload_options(self):

Please write tests there